src/components/DetailsMetrics/DetailsMetrics.js

```diff
@@ -181,24 +315,106 @@ const DetailsMetrics = ({ selectedItem }) => {
+                        >
+                          {resultPercentageDrift.percentageChange}
+                        </div>
+                        <div className="metrics__card-invocation-header_selected_date">
+                          {selectedDate}
+                        </div>
+                        <div className="metrics__card-invocation-header_total_title">Total</div>
+                        <div className="metrics__card-invocation-header_total_score">
+                          {metric.total}
+                        </div>
+                      </div>
+                    </StatsCard.Header>
+                    <div ref={cardRef} className="metrics__card-body">
+                      <div className="metrics__card-invocation-content">
+                        <div className="metrics__card-invocation-content-title">
+                          Endpoint call count
+                        </div>
+                        <div className="metrics__card-invocation-content_container">
+                          <div className="metrics__card-invocation-content_container_drift_icon">
+                            {resultPercentageDrift.positive ? <ArrowUp /> : <ArrowDown />}
+                          </div>
+                          <div
+                            className={`metrics__card-invocation-content_container_${resultPercentageDrift.className}`}
+                          >
+                            {resultPercentageDrift.percentageChange}
+                          </div>
+                          <div>{selectedDate}</div>
+                        </div>
+                        <div className="metrics__card-invocation-content-data">
+                          <div className="metrics__card-invocation-content-data_total_title">
+                            Total
+                          </div>
+                          <div className="metrics__card-invocation-content-data_total_score">
+                            {' '}
+                            {metric.total}
+                          </div>
+                        </div>
+                      </div>
+                      <div className="metrics__card-body-invocation">
+                        <GenericMetricChart
+                          showGrid={expand}
+                          chartConfig={{
+                            gradient: true,
+                            ...gradientConfig,
+                            data: {
+                              labels: metric.labels,
+                              datasets: [
+                                {
+                                  data: metric.points,
+                                  chartType: CHART_TYPE_LINE,
+                                  fill: true,
+                                  metricType: metric.type,
+                                  driftStatusList: [],
+                                  backgroundColor: '#5871F4',
+                                  borderColor: '#5871F4',
```

Comment:
all colors must be in the file `colors.scss`

Reply:
added to export of colors.scss + new branch in dashboard-react-controls